h/static/scripts/app-controller.coffee

```diff
@@ -68,6 +80,8 @@ module.exports = class AppController
         else
           $scope.$emit('annotationDeleted', container.message)
 
+    identity.watch({onlogin, onlogout, onready})
```

Comment:
Suggest you move this up, right under the definitions of `onlog{in, out}...`, so it's immediately clear why they're being defined and what they're used for. Or, frankly, given the size of them:

```coffeescript
identity.watch({
  onlogin: (identity) -> $scope.auth.user = auth.userid(identity)
  onlogout: -> $scope.auth.user = null
  onready: -> $scope.auth.user ?= null
})
```